src/plugin/pid/sched_wrappers.cpp

```diff
@@ -0,0 +1,157 @@
+/* FILE: sched_wrappers.cpp
+ * AUTHOR: Rohan Garg
+ * EMAIL: rohgarg@ccs.neu.edu
+ * Copyright (C) 2015 Rohan Garg
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <sched.h>
+
+#include "virtualpidtable.h"
+#include "dmtcp.h"
+
+#define _real_sched_setaffinity     NEXT_FNC(sched_setaffinity)
+#define _real_sched_getaffinity     NEXT_FNC(sched_getaffinity)
+#define _real_sched_setscheduler    NEXT_FNC(sched_setscheduler)
+#define _real_sched_getscheduler    NEXT_FNC(sched_getscheduler)
+#define _real_sched_setattr         NEXT_FNC(sched_setattr)
+#define _real_sched_getattr         NEXT_FNC(sched_getattr)
+#define _real_sched_setparam        NEXT_FNC(sched_setparam)
+#define _real_sched_getparam        NEXT_FNC(sched_getparam)
```

Comment:
Can we instead follow the pattern from pidwrappers.h/pid_syscallsreal.c? This plugin is weird in that sense. Until we have a chance to do some cleanup, let's try to keep it consistent! :-)